{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -184,8 +184,12 @@
 EMAIL_SUPPORT = get_string('{{ cookiecutter.project_name|upper }}_SUPPORT_EMAIL', 'support@example.com')
 DEFAULT_FROM_EMAIL = get_string('{{ cookiecutter.project_name|upper }}_FROM_EMAIL', 'webmaster@localhost')
 
-MAILGUN_URL = get_string('MAILGUN_URL', 'https://api.mailgun.net/v3/micromasters.mit.edu')
+MAILGUN_URL = get_string('MAILGUN_URL', None)
+if not MAILGUN_URL:
```

Comment:
Could you add `MAILGUN_URL` to `.env.example` too?

Reply:
Yes